src/compiler/sys.ts

```diff
@@ -131,9 +131,6 @@ let pollingChunkSize = createPollingIntervalBasedLevels(defaultChunkLevels);
 export let unchangedPollThresholds = createPollingIntervalBasedLevels(defaultChunkLevels);
 
 function setCustomPollingValues(system: System) {
-    if (!system.getEnvironmentVariable) {

```

Comment:
I used VSCode to search for all implementations of `System` and found only the node `System` defined in this file. `getEnvironmentVariable` is always defined as

```
            getEnvironmentVariable(name: string) {
                return process.env[name] || "";
            },
```

so I felt safe removing this check -- the condition cannot currently evaluate to true.